modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/CallFunction.java

```diff
@@ -228,6 +228,10 @@ protected void setDDMFormFieldValues(
 	private void _extractDDMFormFieldValue(
 		String expression, Map<String, String> parameters) {
 
+		if (expression.isEmpty()) {
+			return;
+		}
```

Comment:
Could we change the condition to use `Validator.isNull(expression)` instead? Or is there a specific reason you need to use `isEmpty`?

Reply:
My goal was to create an early return for the case of the empty string. Is there some better way to validate this case? I believe that null validation will not cover this case. Right?

Reply:
![Screenshot from 2025-05-28 18-21-06](https://github.com/user-attachments/assets/0797052a-ec1d-4458-aa03-cc997d2d2426)

Reply:
I agree! Thanks for the explanation 😄